easybuild/easyconfigs/m/malt/malt-0.4.1.eb

```diff
@@ -0,0 +1,40 @@
+# This is for the custom build of Malt 
+# Malt is downloaded from here:
+# http://ab.inf.uni-tuebingen.de/data/software/malt/download/welcome.html
+# The MALT_unix_0_4_1.sh is executed and the binaries are installed in
+# malt/bin
+# The malt directory is bundled into a tarball which will be installed
+# This is a bit of a hack!
+# The homepage is here:
+# http://ab.inf.uni-tuebingen.de/software/malt/
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'malt'
+version = '0.4.1'
+
+homepage = 'http://ab.inf.uni-tuebingen.de/software/malt/'
+description = """MALT (MEGAN alignment tool) is an extension of MEGAN (metagenome analyzer).
+
+MALT performs alignment of metagenomic reads against a database of reference sequences 
+(such as NR, GenBank or Silva) and produces a MEGAN RMA file as output. The software is 
+currently under development. """
+
+toolchain = {'name': 'dummy', 'version': ''}
```

Comment:
Use of 'dummy' toolchain is deprecated, use 'system' toolchain instead; see https://github.com/easybuilders/easybuild-framework/pull/2877 for more information

Reply:
@sassy-crick Please remove this easyconfig (since you added another using `MALT` as name):

```
eb --update-pr 7626 :malt-0.4.1.eb  --pr-commit-msg "remove malt easyconfig (since name should be uppercase)"